test/cbmc/include/core_http_config.h

```diff
@@ -28,6 +28,32 @@
 #ifndef CORE_HTTP_CONFIG_H
 #define CORE_HTTP_CONFIG_H
 
-/* Empty configuration header simply for compilation. */
+/**
+ * @brief The maximum duration between non-empty network reads while receiving
+ * an HTTP response via the #HTTPClient_Send API function.
+ *
+ * The transport receive function may be called multiple times until the end of
+ * the response is detected by the parser. This timeout represents the maximum
+ * duration that is allowed without any data reception from the network for the
+ * incoming response.
+ *
+ * If the timeout expires, the #HTTPClient_Send function will return
```

Comment:
Can you add a comment to explain the rationale for choosing 1 as the value in the proofs for both the configs?

Reply:
done